Sources/AblyAssetTrackingCore/Configuration/ConnectionConfiguration.swift

```diff
@@ -10,46 +10,92 @@ public enum AuthResult {
     case tokenDetails(TokenDetails)
 }
 
+public class ObjcAuthResultJWT: NSObject {
+    public var value: String
+    
+    public init(value: String) {
+        self.value = value
+    }
+}
+
+public class ObjcAuthResultTokenRequest: NSObject {
+    public var value: TokenRequest
+    
+    public init(value: TokenRequest) {
+        self.value = value
+    }
+}
+
+public class ObjcAuthResultTokenDetails: NSObject {
+    public var value: TokenDetails
+    
+    public init(value: TokenDetails) {
+        self.value = value
+    }
+}
+
 public typealias Token = String
 public typealias AuthCallback = (TokenParams, @escaping (Result<AuthResult, Error>) -> Void) -> Void
+public typealias ObjCAuthCallback = (TokenParams, @escaping (AnyObject?, Error?) -> Void) -> Void
 
 /// A container for connection configuration data used when connecting to Ably
 public class ConnectionConfiguration: NSObject {
     public let apiKey: String?
     public let clientId: String?
     public let authCallback: AuthCallback?
+    public let objcAuthCallback: ObjCAuthCallback?
     
     /**
      Connect to Ably using basic authentication (API Key)
      
      - Parameters:
-       - apiKey: API key string obtained from application dashboard.
-       - clientId: Optional identifier to be assigned to this client.
-       - authCallback: A callback that will be used to authenticate with Ably, including at initial connection and for renewing an expired token.
+     - apiKey: API key string obtained from application dashboard.
```

Comment:
I noticed this indentation change, and this made me think what formatting are we using? DocC/Swift has guidance for this at https://www.swift.org/documentation/docc/writing-symbol-documentation-in-your-source-files, you might be interested :) 

Resolve this conversation whenever you want 😇

Reply:
I use the `conrtol+i` shortcut and the indent is set to 4 spaces / 4 tabs

Reply:
do you think `///` is better than `/** ... */` ?
Is `///` recommended by Apple (or swift community) now?

Reply:
In dart, the linter tells me to use `///`. I think it is a little bit better because each line can be moved around independently. However, DocC says both is supported and **doesn't recommend anything specific**.

However, shouldn't the parameters be children of `- Parameters`?. Maybe Xcode is not good at indenting docs 😅. If we follow the DocC guidelines, we have a better chance of generating DocC as well in the future.

Reply:
here 94caaf50cd1ed4c483d577adf9887374f61d25cb

Reply:
THanks 🚀